### DISCUSSION TITLE
畢業條件查詢網站

### DISCUSSION THREAD

**Discussion by Chattenoire** [1101-GS4719-C/Python-Community-GS4719-C#2211] [General]
不知道大家有沒有發現學校弄了一個新的畢業資格查詢介面，我覺得真的是難得的德政
(不過不知道是不是因為校長換了，比較有用了)

給大家參考一下:
https://cis.ncu.edu.tw/iNCU/academic/graduate/graduateReport

加碼抱怨一下: 這個學校的(校方)高層行政真的都很沒效率，也沒甚麼遠見==

**Comment by Mminniehuang** [comment]
我沒有看過之前的畢業審查系統，不過之前好像有聽學長姐提過很亂很難看（？
去看了一下系統之後覺得壓力好大...離畢業還有一大段距離。馬上就是初選的時間了，要好好再想一下，把該修的課修一修才是上策。

**Comment by Elaine4488** [comment]
其實我覺得還是不太方便看欸，但至少比之前好了。有些可以兩邊抵的課程還是要自己對照一下。

**Comment by poyen0806** [comment]
我還真不知道有這東西，但看到其他人說覺得很不方便
那我還是先繼續自己研究畢業資格好了
如果有時間的話
用python應該蠻好解決的
可以試試看架一個自己的自動化審核網站
